examples/computecloth/computecloth.cpp

```diff
@@ -292,8 +287,7 @@ class VulkanExample : public VulkanExampleBase
 		VkCommandBufferBeginInfo cmdBufInfo = vks::initializers::commandBufferBeginInfo();
 		cmdBufInfo.flags = VK_COMMAND_BUFFER_USAGE_SIMULTANEOUS_USE_BIT;
 
-		for (uint32_t i = 0; i < 2; i++) {
-
+		for (uint32_t i = 0; i < compute.commandBuffers.size(); i++) {
```

Comment:
This feels kind of dumb since it's only a single item array now, but Github failed it's saving throw in producing a reasonable diff when I removed the for loop.